bottlecap/src/logger.rs

```diff
@@ -22,7 +22,12 @@ where
     ) -> fmt::Result {
         // Format values from the event's's metadata:
         let metadata = event.metadata();
-        write!(&mut writer, "DD_EXTENSION | {} | ", metadata.level())?;
+        write!(
+            &mut writer,
+            "DD_EXTENSION | {} | {} | ",
+            metadata.name(),
```

Comment:
This would add the file on every debug log, do we want this?

Reply:
nope